zos-utils/zossrc/AttlsContext.c

```diff
@@ -204,7 +219,7 @@ jstring get_jstring(JNIEnv *env, char* ebcdic, int length)
     if (realSize < length) {
         length = realSize;
     }
-    char *output = (char*) __malloc31(length + 1);
+    char *output = (char*) malloc(length + 1);
```

Comment:
You never check the returned pointer for `NULL` in the entire file. `malloc()` will return `NULL` if it fails to allocate memory.

Reply:
I have added checking of that. Now it throws an exception. I have also improved exception checking.